_dashboards/get-started/quickstart-dashboards.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: default
+title: Quickstart for OpenSearch Dashboards
+nav_order: 20
+has_children: false
+---
+
+# Quickstart for OpenSearch Dashboards
+
+This quickstart covers the core concepts for getting started with OpenSearch Dashboards. You'll learn how to:
+
+- Add sample data
+- Explore and inspect data with **Discover**
+- Visualize data with **Dashboard**
+
+Before you get started, make sure you've installed OpenSearch and OpenSearch Dashboards. For information on installation and configuration, see [Install and configure OpenSearch]({{site.url}}{{site.baseurl}}/install-and-configure/install-opensearch/index/) and [Install and configure OpenSearch Dashboards]({{site.url}}{{site.baseurl}}/install-and-configure/install-dashboards/index/).
+{: .note}
+
+# Add sample data
+
+Sample data sets come with visualizations, dashboards, and more to help you explore Dashboards before you add your own data. To add sample data, perform the following steps:
+
+1. Verify access to OpenSearch Dashboards by connecting to [http://localhost:5601](http://localhost:5601) from a browser. The default username and password are `admin`.
+2. On the OpenSearch Dashboards **Home** page, select **Add sample data**.
+3. Select **Add data** to add the data sets.
+
+![Sample data sets]({{site.url}}{{site.baseurl}}/images/add-sample-data.png)
```

Comment:
```suggestion
<img src="({{site.url}}{{site.baseurl}}/images/add-sample-data.png" alt="Sample data sets">{: .img-fluid}
```

Reply:
This suggestion also applies to any other image links with "!" at the beginning in the document.